com.github.rhyme_player.rhyme.yml

```diff
@@ -0,0 +1,67 @@
+app-id: com.github.rhyme_player.rhyme
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
```suggestion
```

Reply:
This is added to support wayland

Reply:
as said in the review, if it no using wayland by default (ie if the user need to do something) then it doesn't support wayland, therefor the permission should be removed.